src/Components/CreateImageWizard/steps/Packages/Packages.tsx

```diff
@@ -135,6 +136,7 @@ const Packages = () => {
   const { data: epelRepo, isSuccess: isSuccessEpelRepo } =
     useListRepositoriesQuery({
       url: epelRepoUrlByDistribution,
+      origin: ContentOrigin.CUSTOM,
```

Comment:
Sorry, got a bit lost in the discussion so just to check - should this be external only or is this a different case? :sweat_smile:

Reply:
You are correct, not sure why I put custom here, with the filter it shouldn't matter but I'll update regardless.